src/apps/games/models/abstract_game.py

```diff
@@ -130,13 +130,18 @@ class GamesResult(TextChoices):
     def result_text(self):
         if self.resigned:
             return f"{self.winner}+R"
-        if self.winner not in [AbstractGame.GamesResult.BLACK, AbstractGame.GamesResult.WHITE]:
-            return self.winner
+        elif self.winner == AbstractGame.GamesResult.DRAW:
+            return _("Draw")
+        elif self.winner == AbstractGame.GamesResult.NO_RESULT:
+            return _("Void")
 
         if self.winner == AbstractGame.GamesResult.BLACK:
             score = -self.score
-        else:
+        elif self.winner == AbstractGame.GamesResult.WHITE:
             score = self.score
+        else:
```

Comment:
I figure that if we ever decide to add more result types, but we forget to update the code here to translate the result to text, then this will be more reasonable in the meantime before we fix the code, instead of assuming that the score should be translated as if the player were white.